<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Threshold</class>
 <widget class="QDialog" name="Threshold">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Threshold</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>141</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>upper threshold</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="upperThresholdLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>lower threshold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lowerThresholdLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Threshold</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
